### DISCUSSION TITLE
Is it possible to increase exp given by enemies? If not, can it be added?

### DISCUSSION THREAD

**Discussion by nintenfando** [Melledy/Grasscutter#2466] [Q&A] [answered]
Can the experience points given by defeating an enemy changed in any away and if it cannot, is it possible to be added?

**Comment by NotThorny** [comment, marked as answer]
All drops, including exp, are customizable.

If using new drop system, edit `DropTableExcelConfigData.json` in your resources.
If using legacy drops, edit `Drop.json` in your `\Grasscutter\data\` folder.

**Reply by nintenfando** [reply]
Oh, ok thanks!
